sdk/key_vault/src/client.rs

```diff
@@ -75,7 +75,7 @@ impl<'a, T: TokenCredential> KeyVaultClient<'a, T> {
             // Token is valid, return it.
             return Ok(());
         }
-        let resource = format!("https://{}", &self.endpoint_suffix);
+        let resource = format!("https://{}/", &self.endpoint_suffix);
```

Comment:
I wonder if we should check the `endpoint_suffix` for an existing beginning `/` and only add an `/` if one is not already present.

Reply:
That's a good point. I guess a nice way might be to make get_token accept `Url` but I don't know enough about OAUTH2 or this sdk to know if that makes sense. 

As a short stop I've done as you've suggested 👍